cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/AppFabricTestBase.java

```diff
@@ -880,6 +885,29 @@ protected int resumeSchedule(String namespace, String appName, String schedule)
     return doGetSchedules(namespace, schedulesUrl);
   }
 
+  protected List<ScheduleDetail> listSchedulesByTriggerProgram(String namespace, ProgramId programId,
+                                                               co.cask.cdap.api.ProgramStatus... programStatuses)
```

Comment:
You import `co.cask.cdap.proto.ProgramStatus`, but you use `co.cask.cdap.api.ProgramStatus`.

Don't think that is intentional.